.idea/vcs.xml

```diff
@@ -2,5 +2,16 @@
 <project version="4">
   <component name="VcsDirectoryMappings">
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/_includes/community" vcs="Git" />
```

Comment:
I'm not sure what this file is or why it has been modified in this PR, but I don't think it should be in this PR.

Reply:
This is an intelij file
I did not manually edit it
Intelij/the readme's git commands updated this
Just now I reverted it

Reply:
Thanks for fixing.
Is the whole `.idea` directory something controlled by Intelij?
I think we should probably add it to the `.gitignore` file.
(Not in this PR, just in general. A new PR.)

Reply:
Yes it is